### PR TITLE
DOCS-1620: Adds credit card components to Tools

### PR DESCRIPTION
**Pull request by Atiq-MultiSafepay**

### CODE REVIEW

**Review by Jasper-MultiSafepay — CHANGES_REQUESTED** [review]
@Atiq-MultiSafepay 
Is there a way to also include the SVG flowdiagram?

File: content/tools/credit-card-components/credit-card-components.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+After the customer has entered their credit card details, the encrypted data can be sent to the MultiSafepay API to finish the transaction.<br>
+
+On the frontend you can retrieve the encrypted credit card data through the following code:
+
+```
+PaymentComponent.getPaymentData().payment_data.payload
+```
+
+This payload can be sent with a [direct CREDITCARD transaction request]((https://docs.multisafepay.com/api/#create-a-direct-order))
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:129]:
```suggestion
This payload can be sent with a [direct CREDITCARD transaction request](https://docs.multisafepay.com/api/#create-a-direct-order)
```

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/tools/credit-card-components/credit-card-components.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+MultiSafepay provides a [Test environment](/tools/credit-card-components/credit-card-components/#setting-up-your-test-environment) and a [Live environment](/credit-card-components/credit-card-components/#4-push-to-live) <br>
+
+The Test environment is useful for developing and testing the Credit card component integration with our API. Once the integration has been developed, processing real transactions is as simple as changing the API endpoint and using a different API key.<br>
+
+For more information about our API, please visit our [API reference](https://docs.multisafepay.com/api/)
+
+
+### API Authentication
+
+All requests to the MultiSafepay API endpoint require authentication. This is provided by including an API key as an HTTP header in your request. Each website has its own API key so if you are operating multiple websites, be sure to use the correct API key for each site. The API key can be found under the _Website settings_ in your [MultiSafepay Control](https://merchant.multisafepay.com/)<br>
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:21]:
```suggestion
All requests to the MultiSafepay API endpoint require authentication. This is provided by including an API key as an HTTP header in your request. Each website has its own API key so if you are operating multiple websites, be sure to use the correct API key for each site. You can find your API key under the _Website settings_ in your [MultiSafepay Control](https://merchant.multisafepay.com/)<br> 
```

File: content/tools/credit-card-components/credit-card-components.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+### Introduction
+
+The _Payment component_ by MultiSafepay allows the possibility of offering your customers a seamless checkout experience as our Javascript based component is easy to implement in your environment.
+
+
+### Environments
+
+MultiSafepay provides a [Test environment](/tools/credit-card-components/credit-card-components/#setting-up-your-test-environment) and a [Live environment](/credit-card-components/credit-card-components/#4-push-to-live) <br>
+
+The Test environment is useful for developing and testing the Credit card component integration with our API. Once the integration has been developed, processing real transactions is as simple as changing the API endpoint and using a different API key.<br>
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:14]:
```suggestion
The Test environment is useful for developing and testing the Credit card component integration with our API. Once you have developed the integration, processing real transactions is as simple as changing the API endpoint and using your live API key.<br>
```

File: content/tools/credit-card-components/credit-card-components.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+[components.css](https://pay.multisafepay.com/sdk/components/v1/components.css)
+
+{{< alert-notice >}} `<link rel="stylesheet" href="https://testpay.multisafepay.com/sdk/components/v1/components.css">` {{< /alert-notice >}}
+
+
+### Get API token
+
+Get the `api_token`, which is used for encrypting the credit card input. This call uses authentication, so **do not expose your own** `api_key`. Make sure your server sends this request and not the client or browser. For example, you can load this when the customer is loading the checkout of your website.
+
+`GET /v1/connect/auth/api_token`
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:47]:
```suggestion
`GET https://api.multisafepay.com/v1/connect/auth/api_token`
```
Specify full API endpoint

File: content/tools/credit-card-components/credit-card-components.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+```
+## Implementation
+
+### 1. Initialize Payment Components library
+
+You must initialize the payment components library and link it to the container element (selector).
+
+The constructor takes three values:
+
+`env` -> test/live, decides the API mode<br>
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:66]: This doesn't work
```suggestion
`env` -> test/live, decides the API mode {{< br >}} 
```

File: content/tools/credit-card-components/credit-card-components.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+## Implementation
+
+### 1. Initialize Payment Components library
+
+You must initialize the payment components library and link it to the container element (selector).
+
+The constructor takes three values:
+
+`env` -> test/live, decides the API mode<br> 
+`apiToken` -> api_token from step 1<br>
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:67]:
```suggestion
`apiToken` -> api_token from step 1 {{< br >}} 
```

File: content/tools/credit-card-components/credit-card-components.md [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+
+After completion, the customer will be returned to the `redirect_url` from the transaction request.
+
+### 4. Push to Live
+
+#### Live environment files
+
+After testing, you may proceed to change the Test files to the Live files listed below:
+
+* components.js<br>
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:142]:
```suggestion
**components.js**
```

File: content/tools/credit-card-components/credit-card-components.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+
+ {{< alert-notice >}} `<script src="https://pay.multisafepay.com/sdk/components/v1/components.js"></script>` {{< /alert-notice >}}
+
+* components.css<br>
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:146]:
```suggestion
**components.css**
```

File: content/tools/credit-card-components/credit-card-components.md [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+});
+```
+<br>
+## Optional features
+### CSS styling
+
+Our CSS is designed to give you full styling control over the component. Below is an example of the main classes of our component:
+
+{{< br >}}
+{{< responsive_svg src="/diagrams/svg/CCC_CSS" alt="TEST" align="center" >}}
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:168]: This breaks on mobile, put it live now but will have to fix later.

File: content/tools/credit-card-components/credit-card-components.md [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+After completion, the customer will be returned to the `redirect_url` from the transaction request.
+
+### 4. Push to Live
+
+#### Live environment files
+
+After testing, you may proceed to change the Test files to the Live files listed below:
+
+* components.js<br>
+
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:143]:
```suggestion
```

File: content/tools/credit-card-components/credit-card-components.md [hunk 11]
```diff
@@ -0,0 +1,213 @@
...
+ {{< alert-notice >}} `<script src="https://pay.multisafepay.com/sdk/components/v1/components.js"></script>` {{< /alert-notice >}}
+
```
Comment (Kris-MultiSafepay) [inline content/tools/credit-card-components/credit-card-components.md:145]:
```suggestion
```

**Review by Kris-MultiSafepay — APPROVED** [review]

**Review by Kris-MultiSafepay — APPROVED** [review]

**Pull request merged**